Procfile

```diff
@@ -1,2 +1,2 @@
-web:        babel-node bin/main.js server
-scheduler:  babel-node bin/main.js scheduler
\ No newline at end of file
+web:        babel-node src/main.js server
```

Comment:
Oh, and also, this behaviour is depending on babel-node being installed, but that binary was removed (iirc) in an update to the 'babel' package a while ago, so this would likely fail on deploy as the required binary would be absent.